app/javascript/src/Common/PlanSelector.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { TextInput } from '@patternfly/react-core'
+
+import './PlanSelector.scss'
+
+class PlanSelector extends React.Component {
+
+  state = {
+    searchTerm: '',
+    showPlans: false,
+    plans: this.props.plans,
+    selectedPlan: { }
+  }
+
+  onTextChange = (searchTerm) => {
+    const filteredPlans = this.props.plans.filter(plan => plan.name.toLowerCase().indexOf(searchTerm.toLowerCase()) > -1)
```

Comment:
Wasn't select2 providing also some fuzzy search?

Also, how this handles hundreds of plans that some customers can have?

Reply:
What kind of fuzzy search? Isn't it enough with a filter by name?

Reply:
That kind that where `pc` would match `Plan Custom`. But from looking at GH it does not do that, so we are not dropping any functionality: https://github.com/select2/select2/issues/4837